_wiki/sfmb_betatest_faq.md

```diff
@@ -56,6 +56,22 @@ latex   : false
 - Because game uses Discord Rich Presence to check your Discord ID.
 - It's best way to check user is REAL beta tester in server or not.
 
+### Q. Some game sprites are bugged, what should i do?
+- Game sprites got corrupted by some reason, mostly download issue. Try [This](sfmb://forceupdate)
+- Or put `sfmb://forceupdate` address on your browser. It should fix corrupted files.
+
+### Q. A friend wants to join the server. How can i invite them?
+- You can talk with an Inviter, but is not guaranteed that your friend will be accepted.
+
+### Q. How can i get Beta Tester Lvl2?
+- Beta Tester Lvl2 is a complex role, you should have knowledge on what a Beta Tester should do in game development, how Beta Testing works. This role can't be obtained with asking, this role is obtained with effort and Neo will choose who deserves it.

```

Comment:
Long sentences like this don't suit the wiki.
Multiple concise sentences are better.